plugins/inputs/haproxy/sample.conf

```diff
@@ -1,19 +1,23 @@
-# Read metrics of HAProxy, via socket or HTTP stats page
+# Read metrics of HAProxy, via stats socket or http endpoints
 [[inputs.haproxy]]
-  ## An array of address to gather stats about. Specify an ip on hostname
-  ## with optional port. ie localhost, 10.10.3.33:1936, etc.
-  ## Make sure you specify the complete path to the stats endpoint
-  ## including the protocol, ie http://10.10.3.33:1936/haproxy?stats
-
-  ## Credentials for basic HTTP authentication
-  # username = "admin"
-  # password = "admin"
-
-  ## If no servers are specified, then default to 127.0.0.1:1936/haproxy?stats
+  ## List of stats endpoints. Metrics can be collected from both http and socket
+  ## endpoints. Examples of valid endpoints:
+  ##   - http://myhaproxy.com:1936/haproxy?stats
+  ##   - https://myhaproxy.com:8000/stats
+  ##   - socket:/run/haproxy/admin.sock
+  ##   - /run/haproxy/*.sock
+  ##   - tcp://127.0.0.1:1936
+  ##
+  ## Server addresses not starting with 'http://', 'https://', 'tcp://' will be
+  ## treated as possible sockets.
+  ##
+  ## When specifying local socket, glob patterns are supported.
+  ##
+  ## If no servers are specified, then defaults to 127.0.0.1:1936/haproxy?stats
   servers = ["http://myhaproxy.com:1936/haproxy?stats"]
 
   ## You can also use local socket with standard wildcard globbing.
-  ## Server address not starting with 'http' will be treated as a possible
+  ## Server address not starting with 'http://', 'https://', 'tcp://' will be treated as a possible
   ## socket, so both examples below are valid.
   # servers = ["socket:/run/haproxy/admin.sock", "/run/haproxy/*.sock"]
```

Comment:
Can you get rid of this second section and run `make docs` please. Thanks!

Reply:
This was done in a previous commit